ui/lib/core/stories/list-view.stories.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/extensions */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import notes from './list-view.md';
+import ArrayProxy from '@ember/array/proxy';
+
+let filtered = ArrayProxy.create({ content: [] });
```

Comment:
out of curiosity, why'd you have to use `ArrayProxy` here?

Reply:
The component usually takes a ember data model array which can have meta information on it and we use that meta information to know if we should show pagination, but also to be able to tell the difference between an empty list and a list that is empty because of an overly-specific filter. Maybe this should change to use ModelRecordArrary from ember-data instead 🤔

Reply:
I looked into it and using RecordArray directly is kinda a pain, but ArrayProxy is the base class of RecordArray so this is probably good enough for now.